python/neutron-understack/neutron_understack/nautobot.py

```diff
@@ -98,12 +100,14 @@ def make_api_request(
     def ucvni_create(
         self,
         network_id: str,
+        project_id: str,
```

Comment:
```suggestion
        project_id: UUID,
```

Is what I meant in my review. But we need to do that for all arguments.

Reply:
we cannot send UUID type attributes directly into payload we need to convert back into string otherwise we get "UUID is not JSON serializable" error.
Removing the conversion logic completely, since Django can handle it.

Reply:
Right. Didn't mean we send it directly. Let's get this merged and we can mess with it afterwards.